### PR TITLE
Improved/Added French translation

### PR DESCRIPTION
**Pull request by lionirdeadman**

### CODE REVIEW

File: src/locales/fr.json [hunk 1]
```diff
@@ -96,6 +100,7 @@
     "de": "Allemand",
     "el": "Grec",
     "en-GB": "Anglais (Royaume Uni)",
+    "en-GB": "Anglais (États-Unis)",
```
Comment (7coil) [inline src/locales/fr.json]: You managed to copy the en-GB key twice

File: src/locales/fr.json [hunk 2]
```diff
@@ -122,7 +128,6 @@
     "bots": {
       "backgroundAlt": "Logo pour ls.terminal.ink",
       "description": "Une boutique d'applications pour des bots Discord",
-      "inMyLanguage": "Bots dans ma langue",
```
Comment (7coil) [inline src/locales/fr.json]: This key should exist

File: src/locales/fr.json [hunk 3]
```diff
@@ -379,18 +398,12 @@
   },
   "site": {
     "name": "Liste de bots Discord par ls.terminal.ink",
```
Comment (7coil) [inline src/locales/fr.json]: "discord apps"

**Comment by lionirdeadman** [comment]
All of the review comments were resolved, sorry about that :sweat:

**Pull request merged**